entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/objectbuilder/ChainingObjectBuilder.java

```diff
@@ -48,19 +47,12 @@ public ChainingObjectBuilder(TupleTransformatorFactory transformatorFactory, Obj
     }
 
     @Override
-    @SuppressWarnings("unchecked")
     public T build(Object[] tuple) {
-        return (T) tuple;
+        return objectBuilder.build(transformator.transform(tuple));
```

Comment:
Is a similar fix also desired for [`LateAdditionalObjectBuilder`](https://github.com/Blazebit/blaze-persistence/blob/aca8349abd3605a56a87c6d126c146795d791848/entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/objectbuilder/LateAdditionalObjectBuilder.java) and sorts?

Reply:
That object builder is only used for subselect queries that fetch an attribute, but I think what we could do is modelling a SELECT with batch size 1 and subselect/multiset-fetch as normal TupleTransformer instead of as TupleListTransformer which allows more use cases to use streaming. I'll look into this.